docker/Dockerfile

```diff
@@ -1,27 +1,31 @@
-FROM jupyter/base-notebook:python-3.8.6
-LABEL maintainer="pyvista Developers"
+FROM jupyter/base-notebook:python-3.9.7
+LABEL maintainer="PyVista Developers"
 
 USER root
 RUN apt-get update \
  && apt-get install  -yq --no-install-recommends \
     libgl1-mesa-glx \
     libglu1-mesa \
```

Comment:
We don't need these with the custom OSMesa VTK wheel, right?

Reply:
Correct. Removed in https://github.com/pyvista/pyvista/pull/2011/commits/9cb0524b3cdac36a36350822342bf4dcc4995267.